tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -1255,3 +1255,89 @@ TEST(caffe2, EQ1D) {
   // Graph has two inputs and one output.
   EXPECT_EQ(mod.getPlaceholders().size(), 3);
 }
+
+// Test loading a SparseToDense operator.
+TEST(caffe2, sparseToDense) {
```

Comment:
If I was implementing this, I would not add a test to `caffe2ImporterTest` at all. Of course, it's better to have more tests. But we are mixing tests here a bit.

This test runs the net and checks results. While the other new unit test in `OperatorTest` does the same, but with different input. Ideally all "run and check result" tests should be in the same place. And Importer tests should only check Importer's logic, which is complicated in some cases, but not always. For `SparseToDense`, the logic in Importer is trivial, so I'd not bother adding a file specifically for this.

What do you think? Maybe just check node's type in this test?

Reply:
Yes, I agree. I will check only the structure of the graph here and move the "run and check result" test to the operator tests.